_rules/image-accessible-name-descriptive-qt1vmo.md

```diff
@@ -0,0 +1,295 @@
+---
+id: qt1vmo
+name: Image accessible name is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the accessible names of images describe their topic or purpose.
+accessibility_requirements:
+  wcag20:1.1.1: # Non-Text Content (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G94: # Providing short text alternative for non-text content that serves the same purpose and presents the same information as the non-text content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G95: # Providing short text alternatives that provide a brief description of the non-text content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - DOM Tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Wilco Fiers
+  assets:
+    - W3C; HTML and W3C logo
+    - Wikimedia; Fireworks image
+    - Adobe; PDF logo
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'alt-require'
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+Any `img`, `canvas` or `svg` element that is [visible][] and has a non-empty [accessible name][], except if one of the following is true:
+
+- The element has an [ancestor][] in the [flat tree][] that is [named from author][]; or
+- The element is an `img` element where the [current request][]'s [state][image request state] is not [completely available][].
+
+## Expectation
+
+Each test target has an [accessible name][] that serves an equivalent purpose to the [non-text content][] of that test target.
```

Comment:
I'm a bit concerned with the difference between the expectation of this rule "... serves an equivalent purpose to ...." and the description of this rule + all the test cases examples of this rule "... describe their topic or purpose". For example, passed example 1 with the alternative text "W3C" **serve as an equivalent purpose** for the W3C logo or **describe its purpose**?
If both these statements reflect the same meaning that we should use one consistent statement in this rule.

Reply:
I've updated the description. I think this resolves it.